packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -1199,14 +1198,16 @@ If more than 100 groups match the filter, then the request fails. Expect that th
-| `code `                     | N/A                                                                                                  | N/A                                         |
-| `token`                     | N/A                                                                                                  | N/A                                         |
-| `id_token`                  | Claims associated with requested scopes.                                                             | N/A                                         |
-| `id_token` `code `          | Claims associated with requested scopes.                                                             | N/A                                         |
-| `id_token` `token`          | `email` if email scope is requested; `name` and `preferred_username` if profile scope is requested   | Claims associated with the requested scopes |
-| `code` `id_token` `token`   | `email` if email scope is requested; `name` and `preferred_username` if profile scope is requested   | Claims associated with the requested scopes |
+  | Response Type               | Claims Returned in ID Token                                                                          | Claims Returned from the Userinfo Endpoint  |
+  | :-------------------------- | :--------------------------------------------------------------------------------------------------- | :------------------------------------------ |
+  | `code `                     | N/A                                                                                                  | N/A                                         |
+  | `token`                     | N/A                                                                                                  | N/A                                         |
+  | `id_token`                  | Claims associated with requested scopes.                                                             | N/A                                         |
+  | `id_token` `code `          | Claims associated with requested scopes.                                                             | N/A                                         |
+  | `id_token` `token`          | `email` if email scope is requested; `name` and `preferred_username` if profile scope is requested   | Claims associated with the requested scopes |
+  | `code` `id_token` `token`   | `email` if email scope is requested; `name` and `preferred_username` if profile scope is requested   | Claims associated with the requested scopes |
+
+> Note: When using the Custom Authorization Server, whether a claim is returned depends on the value set for the [`alwaysIncludeinToken` property](/docs/reference/api/authorization-servers/#claim-properties) in the claim configuration. If `alwaysIncludeinToken` is set to `True`, the claim is included. If the value is set to `False`, the claim isn't included.
```

Comment:
Whether the claims are included in the id_token or not still depends on the scope and response type. So I wonder if we should add another column "Claims Returned ID Token for Custom AS", and for _`id_token` `token`  and `code` `id_token` `token`_, the description will be _Claims associated with requested scopes and `alwaysIncludeinToken` property is true_. If the table become too wide to fit the page, we can remove the column of userInfo, just add a description that it will always return all the claims associated with the requested scopes.